File: backend/Models/Fear.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace backend.Models;

public class Fear
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int Id { get; set; }

    public string Title { get; set; } = string.Empty;
    public string? Description { get; set; }
    public int CurrentAnxietyLevel { get; set; }
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

    public List<ExposureSession> ExposureSessions { get; set; } = new();
}

File: frontend/src/types/Fear.ts
export type ExposureSession = {
  id: number;
  fearId: number;
  date: string;
  anxietyBefore: number;
  anxietyAfter: number;
  notes?: string;
};

export type Fear = {
  id: number;
  title: string;
  description?: string;
  currentAnxietyLevel: number;
  createdAt: string;
  exposureSessions: ExposureSession[];
};